packages/@adobe/spectrum-css-temp/components/calendar/index.css

```diff
@@ -205,64 +216,61 @@ governing permissions and limitations under the License.
     }
   }
 
+  /* hover and selected states */
+  &:after {
+    position: absolute;
+    top: calc(50% - calc(var(--spectrum-calendar-day-width) / 2));
+    left: calc(50% - calc(var(--spectrum-calendar-day-width) / 2));
+
+    display: block;
+
+    height: var(--spectrum-calendar-day-height);
+    width: var(--spectrum-calendar-day-width);
+
+    border-radius: var(--spectrum-calendar-day-width);
+
+    content: '';
+    z-index: -1;
+  }
+
```

Comment:
Not sure if this is the reason why, but noted that the Calendar stories don't have the blue circle on the selected date making it really hard see the date number:
![image](https://user-images.githubusercontent.com/9661127/139754148-83a6e384-db20-457a-941f-dd11d9f50dbd.png)

Doesn't seem to be an issue in the date pickers

Reply:
Whoa weird issue! Seems to be caused by the `z-index: -1` on the `:after` element. Adding `isolation: isolate` to the calendar (which is already there in the popover) seems to fix it.

Reply:
Should be fixed now.